### ISSUE TITLE
Dependencies installation

### ISSUE THREAD

**Issue by jonesmz** [BearishSun/BansheeEngine#184]
From here: https://github.com/BearishSun/BansheeEngine/blob/master/Documentation/GitHub/compiling.md

DirectX Debug Layer (Required by default on Windows 10)
Optional if you have choosen a different RenderAPI in CMake options
Go to Settings panel (type "Settings" in Start)->System->Apps & features->Manage optional Features->Add a feature->Select "Graphics Tools"

There's no such option?

Perhaps Microsoft has removed it since this document was written, or there's a step missing for how to enable it?

I do not have windows set up with any developer mode style options. Perhaps that's the problem?

**Comment by BearishSun** [comment]
I'm able to find it on my machine (except the "Graphics Tools" feature is already listed as installed). I don't think any kind of developer mode is needed. At which step in particular is an option missing? "Graphics Tools" in the optional feature list?

**Comment by jonesmz** [comment]
Yes, sorry. "Graphics Tools" isn't in the list of features in the "Add a feature" screen.

**Comment by BearishSun** [comment]
It's possible they have moved it to Windows 10 SDK. Or perhaps VS2017 ships with it by default now.

Windows 10 SDK would be my first guess. Without Graphics Tools the app should fail to launch saying something about debug functionality not working. However it should run fine without this feature in Release mode - so you could determine if it's /somehow/ installed.
